tests/test-animation.py.in

```diff
@@ -42,8 +52,22 @@ def collectKeyFrames(action, bone_name=''):
             keyframe_nb.setdefault('scale', []).append(len(cu.keyframe_points))
         if  bone_name != '' and 'rotation_quaternion' in cu.data_path or cu.data_path == 'rotation_quaternion':
             keyframe_nb.setdefault('rotation_quaternion', []).append(len(cu.keyframe_points))
+
     return keyframe_nb
 
+def collectOsgMorphKeyframes(osganimation):
+    targets = {}
+    for channel in osganimation.channels:
+        targets[channel.target] = [key for key in channel.keys]
```

Comment:
`targets = {channel.target: list(channel.keys) for channel in osganimation.channels}`